src/lib/gundb/UserStorageClass.js

```diff
@@ -1630,31 +1651,22 @@ export class UserStorage {
       })
   }
 
-  /**
-   * Sets the event's otpl status
-   * @param {string} eventId
-   * @param {string} status
-   * @returns {Promise<FeedEvent>}
-   */
-  async updateEventOtplStatus(eventId: string, status: string): Promise<FeedEvent> {
-    const feedEvent = await this.getFeedItemByTransactionHash(eventId)
-    feedEvent.status = status
-
-    if (feedEvent.data) {
-      feedEvent.data.otplStatus = status
-    }
-
-    return this.updateFeedEvent(feedEvent)
-  }
-
   /**
    * Sets the event's status as error
    * @param {*} err
    * @returns {Promise<FeedEvent>}
    */
   async markWithErrorEvent(err: any): Promise<FeedEvent> {
-    const error = JSON.parse(`{${err.message.split('{')[1]}`)
-    await this.updateEventOtplStatus(error.transactionHash, 'error')
+    const release = await this.feedMutex.lock()
+
+    try {
+      const error = JSON.parse(`{${err.message.split('{')[1]}`)
+      await this.updateEventStatus(error.transactionHash, 'error')
```

Comment:
verify transactionhash is defined

Reply:
@sirpy it is defined. I have already checked it before.

Reply:
@yaroslav-fedyshyn-nordwhale its not defined in case the error json you parse simply doesnt contain it

Reply:
Done